charts/rancher-k3s-upgrader/0.2.0/questions.yml

```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.4.0-rc1
```

Comment:
Is this only applicable for 2.5? If so, it should be 2.5.0-alpha1

Reply:
Maybe I don't know what I'm talking about but this was originally intended for ECM only (ember ui). I do not know if it would work in R 2.5 (dashboard).

Reply:
no, it's applicable for 2.4 also, so this version of SUC fixes the issue where the number of jobs created sometimes exceeds concurrency.

Reply:
@davidnuzik These charts are only available and applicable to ember UI. 

@MonzElmasry As long as these can work in 2.4 as well, then I am comfortable with the Rancher min version. Has someone in QA tested upgrade situations using this chart or a setup with both versions available?

Reply:
it was tested here
https://github.com/rancher/rancher/issues/27724
the rke2-upgrade issue is related to rke2 but SUC version worked fine.

Reply:
@deniseschannon what else is needed to push this through and get this merged in?